schema/randomprime.schema.json

```
@@ -2535,6 +2536,12 @@
                 "flamethrower": {
                     "type": "boolean",
                     "default": false
+                },
+                "unknownItem2": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "maximum": 2147483647,
+                    "default": 12
                 }
```

I've decided I don't want the JSON user to have to do any bitwise math. I'd rather we expand our definition of "inventory" to include the custom items, and then do the bitwise math in randomprime when updating game objects.

Feel free to leave `unknownItem2` but it should throw an error if the user uses both. If you're feeling helpful, I'd appreciate it if you could add in `powerSuit` and `unknownItem1` while you're at it (that bit isn't stritcly necessary for this PR, however)